rcsbapi/const.py

```diff
@@ -16,10 +15,6 @@ class Const:
     # Search API constants
     STRUCTURE_INDEX: int = 0
     CHEMICAL_INDEX: int = 0
-    STRUCTURE_ATTRIBUTE_SCHEMA_URL: str = "http://search.rcsb.org/rcsbsearch/v2/metadata/schema"
-    STRUCTURE_ATTRIBUTE_SCHEMA_FILE: str = "resources/metadata_schema.json"
-    CHEMICAL_ATTRIBUTE_SCHEMA_URL: str = "https://search.rcsb.org/rcsbsearch/v2/metadata/chemical/schema"
-    CHEMICAL_ATTRIBUTE_SCHEMA_FILE: str = "resources/chemical_schema.json"
     SEARCH_SCHEMA_URL: str = "https://search.rcsb.org/schema/search/request/json-schema-rcsb_search_query.json"
```

Comment:
I know this isn't used anywhere (yet), but might as well be more specific and consistent with the name:
```suggestion
    SEARCH_API_REQUEST_SCHEMA_URL: str = "https://search.rcsb.org/schema/search/request/json-schema-rcsb_search_query.json"
```